cypress/component/rich-text/RichTextEditor.EmbeddedResourceBlocks.spec.ts

```diff
@@ -1,52 +1,33 @@
-/* eslint-disable mocha/no-setup-in-describe */
-
 import { BLOCKS } from '@contentful/rich-text-types';
 
 import { block, document as doc, text } from '../../../packages/rich-text/src/helpers/nodeFactory';
-import { getIframe } from '../../fixtures/utils';
+import { mod } from '../../fixtures/utils';
+import { emptyParagraph, KEYS, paragraphWithText } from './helpers';
 import { RichTextPage } from './RichTextPage';
+import { mountRichTextEditor } from './utils';
 
 // the sticky toolbar gets in the way of some of the tests, therefore
 // we increase the viewport height to fit the whole page on the screen
 
 describe('Rich Text Editor - Embedded Resource Blocks', { viewportHeight: 2000 }, () => {
   let richText: RichTextPage;
-
-  // copied from the 'is-hotkey' library we use for RichText shortcuts
-  const IS_MAC =
-    typeof window != 'undefined' && /Mac|iPod|iPhone|iPad/.test(window.navigator.platform);
-  const mod = IS_MAC ? 'meta' : 'control';
-  const buildHelper =
-    (type) =>
-    (...children) =>
-      block(type, {}, ...children);
-  const paragraph = buildHelper(BLOCKS.PARAGRAPH);
-  const paragraphWithText = (t) => paragraph(text(t, []));
-  const emptyParagraph = () => paragraphWithText('');
   const expectDocumentToBeEmpty = () => richText.expectValue(undefined);
+
   const resourceBlock = () =>
     block(BLOCKS.EMBEDDED_RESOURCE, {
       target: {
         sys: {
-          urn: 'crn:contentful:::content:spaces/space-id/entries/example-entity-urn',
+          urn: 'crn:contentful:::content:spaces/indifferent/entries/published-entry',
```

Comment:
Nit: why is the space id called `indifferent`? feels weird every time I read it. Maybe `another-space-id` or `cross-space-id` or something in that spirit?

Reply:
I agree, but I don't know. That's how it was in fixtures and it will be expensive to change that, unfortunately, because of encoded base64 strings somewhere in the tests.